khelo/lib/ui/flow/profile/components/complete_profile_progress.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:data/api/user/user_models.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:khelo/domain/extensions/context_extensions.dart';
+import 'package:style/extensions/context_extensions.dart';
+import 'package:style/text/app_text_style.dart';
+
+class CompleteProfileProgress extends ConsumerStatefulWidget {
+  final UserModel? user;
+
+  const CompleteProfileProgress({super.key, this.user});
+
+  @override
+  ConsumerState<CompleteProfileProgress> createState() =>
+      _CompleteProfileProgressState();
+}
+
+class _CompleteProfileProgressState
+    extends ConsumerState<CompleteProfileProgress> {
+  @override
+  Widget build(BuildContext context) {
+    if (widget.user?.profile_img_url != null &&
+        widget.user?.gender != null &&
+        widget.user?.player_role != null &&
+        widget.user?.batting_style != null &&
+        widget.user?.bowling_style != null) return const SizedBox();
```

Comment:
if we're showing prompt for these fields only, then progress also should be measured for these fields only.

Reply:
No because we need to show the required field progress as well.